tests/infrastructure/test_azure.py

```
@@ -0,0 +1,1883 @@
+        mock_client = MagicMock()
+        azure_provider.ssh_client = mock_client
+
+        mock_stdout = MagicMock()
+        mock_stdout.read.return_value = b""
+        mock_stdout.channel.recv_exit_status.return_value = 1
+        mock_stderr = MagicMock()
+        mock_stderr.read.return_value = b"error\n"
+
+        mock_client.exec_command.return_value = (None, mock_stdout, mock_stderr)
+
+        exit_code, stdout, stderr = await azure_provider._ssh_exec("false")
```

Ruff (0.14.14)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)